.gitignore

```diff
@@ -5,4 +5,5 @@ __pycache__/
 venv
 .eggs
 .pytest_cache
-*.egg-info
\ No newline at end of file
+*.egg-info
+build
```

Comment:
this was created as a result of `pip install .` - if that's not the correct way to work locally, then we can remove.